homeassistant/components/zone/__init__.py

```diff
@@ -373,6 +375,7 @@ def _generate_attrs(self) -> None:
             ATTR_LONGITUDE: self._config[CONF_LONGITUDE],
             ATTR_RADIUS: self._config[CONF_RADIUS],
             ATTR_PASSIVE: self._config[CONF_PASSIVE],
+            ATTR_ENTITY_ID: list(self._persons_in_zone),
```

Comment:
The attribute name is confusing to me. I would assume it to be the zone entity id if I didn't know the code. I would simply name it `"persons"` or `"persons_in_zone"`.

Reply:
That is to make it consistent with groups an allow it to work with the expand methods of our templating

Reply:
Adjusted.